.gitignore

```diff
@@ -20,10 +20,13 @@
 .env.development.local
 .env.test.local
 .env.production.local
+untracked
+docker-compose.yml
+docker-dev
```

Comment:
The docker setup I use is a bit custom, it expects nginx-proxy and uses debian instead of alpine, which I prefer for development. Happy to share but it might not fit everyone's environment / preference.
[Here's what I'm using](https://gist.github.com/ttoomey/c6acecc5c22e6087cf76f95c68e5e985)

I can make another PR for this docker config if it would be helpful

Reply:
Cool, thanks for sharing. I think there's no need for a PR now but it is great to know that you have a working setup!